ios/brave-ios/Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -96,17 +147,14 @@ struct SubmitReportView: View {
         Button(
           Strings.Shields.reportBrokenSubmitButtonTitle,
           action: {
-            isSubmittingReport = true
-
             Task { @MainActor in
-              await createAndSubmitReport()
-
-              isSubmitted = true
-              try await Task.sleep(seconds: 4)
-              dismiss()
+              if await createAndSubmitReport() {
+                try await Task.sleep(seconds: 4)
```

Comment:
sleep 4 seconds seems a very long time lol

Reply:
Yeah I’m not sure if this was requirement or delay so the report has time to submit without releasing the underlying Webcompat Reporter. Just moved existing code, I can test if it’s required at all.

Reply:
This is just a timer to auto-dismiss the view after we show the successfully submitted view state. I think it's fine to leave it, user can still manually dismiss earlier than that by swiping.